content/blog/2020-10-12-october-20-dvc-heartbeat.md

```diff
@@ -0,0 +1,162 @@
+---
+title: October '20 Heartbeat
+date: 2020-10-12
+description: |
+  This month, hear about our international talks, new video docs, 
```

Comment:
is there one specific we want to emphasize? or Youtube channel? This section is important to for SEO and when people read, so it's better to utilize it.